CMakeLists.txt

```diff
@@ -66,16 +69,27 @@ include_directories(include)
 
 ## Install executable
 install(TARGETS ${PROJECT_NAME}
-    ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-    LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-    RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
+    ARCHIVE DESTINATION lib
+    LIBRARY DESTINATION lib
+    RUNTIME DESTINATION bin
+    PUBLIC_HEADER DESTINATION include/${PROJECT_NAME}
 )
 
 ## Install headers
 install(FILES include/serial/serial.h include/serial/v8stdint.h
-  DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}/serial)
+    DESTINATION include/serial)
+
+## Install CMake config
+install(FILES cmake/serialConfig.cmake
+    DESTINATION share/serial/cmake)
+
+
+## Install package.xml
+install(FILES package.xml
+    DESTINATION share/serial)
```

Comment:
This might be a good time to change to exported targets, and also consider putting the headers in a separate subfolder, which we've been doing in ROS 2 for a while now to promote better header isolation in a merged install space.

I.e. we'd put the headers in `include/serial/serial` and the exported include flags would be like `-I/path/to/include/serial` with the code still doing `#include "serial/serial.h"`.